fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -405,6 +414,58 @@ public CommandProcessingResult withdrawal(final Long savingsId, final JsonComman
                 .build();
     }
 
+    @Transactional
+    @Override
+    public CommandProcessingResult withdrawToLinkedAccount(final Long savingsId, final JsonCommand command) {
+
+        boolean isPaymentHubIntegrationEnabled = configurationDomainService.isPaymentHubIntegrationEnabled();
+        if (!isPaymentHubIntegrationEnabled) {
+            throw new GlobalConfigurationNotEnabledException("enable-payment-hub-integration");
+        }
+
+        this.savingsAccountTransactionDataValidator.validate(command);
+
+        final LocalDate transactionDate = command.localDateValueOfParameterNamed("transactionDate");
+        final BigDecimal transactionAmount = command.bigDecimalValueOfParameterNamed("transactionAmount");
+
+        final Locale locale = command.extractLocale();
+        final DateTimeFormatter fmt = DateTimeFormatter.ofPattern(command.dateFormat()).withLocale(locale);
+
+        final Map<String, Object> changes = new LinkedHashMap<>();
+        final PaymentDetail paymentDetail = this.paymentDetailWritePlatformService.createAndPersistPaymentDetail(command, changes);
+
+        final boolean backdatedTxnsAllowedTill = this.savingAccountAssembler.getPivotConfigStatus();
+
+        final SavingsAccount account = this.savingAccountAssembler.assembleFrom(savingsId, backdatedTxnsAllowedTill);
+
+        InteropIdentifier identifier1 = interopService.getIndentifierByAccount(account);
```

Comment:
Please dont use `identifier1` variable. It is not helpful and not descriptive at all.